config/packages/app_additional_contact_data_admin.yaml

```diff
@@ -0,0 +1,5 @@
+sulu_admin:
+    resources:
+        additional_contact_data:
```

Comment:
would add this to the resources under `config/packages/sulu_admin.yaml` or what do you think?

https://github.com/sulu/sulu-demo/blob/98453fc83c2ea9f068af58cb5eda49dfba6ee8af/config/packages/sulu_admin.yaml#L9

Reply:
we did it like this in #84 to make the file easier to copy from the example PR. think i would leave it like this for that reason 🙂

Reply:
is okay for me maybe we maybe we add a comment here in the PR that it could also be added to the `sulu_admin.yaml`

Reply:
is okay for me - feel free to add the comment! will remove this comments in a few minutes to keep the PR clean 😇